serverest/tests/commons/Commons.robot

```diff
@@ -0,0 +1,23 @@
+***Keywords***
+Create Users
```

Comment:
@RafaelFelice this KW doesn't "create" anything.
I didn't understand the usage.

Could you explain to me, please? thanks.